README.md

```diff
@@ -68,14 +73,51 @@ const playbackId = await Video.Assets.createPlaybackId(asset.id, {
 
 Or, if you don't have the files online already, you can ingest one via the direct uploads API.
 
+However, before that, a word of warning on using request libraries.
+
+### Request Library Usage in Node
+
+Since 2019, `request` has gone into maintenance mode. 
+
+We must use an alternate request library such as `node-fetch`.
+
+`node-fetch` supports both CommonJS (`require` syntax) and ESM (`import` syntax). 
+
+[With v3, you must use ESM only. For CommonJS, use v2.](https://github.com/node-fetch/node-fetch#commonjs)
```

Comment:
this is a little misleading for novices - transpilers will add ESM shims even for CJS use (and not too many people write vanilla JS these days!). Might go away if we just have a require vs import version of the docs.

Reply:
Perhaps we could precede with, "Unless you're using a transpiler"? The explicit callout _could_ help prevent the friction point of users installing a certain version of `node-fetch`- likely the newest - which `node-fetch` explicitly calls out as an ESM-only module - and not knowing why `require` doesn't work 🤔

Reply:
Maybe. I don't have a super strong feeling here. @jsanford8? @jaredsmith?

Reply:
Made separate examples using either syntax. I guess it depends if we think it's our responsibility to reduce friction on a package that we recommend they use. Open to either. We could just cut the callout. `node-fetch` does it for us.